src/validationinterface.cpp

```diff
@@ -183,6 +183,12 @@ void ValidationSignals::UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlo
                           fInitialDownload);
 }
 
+void ValidationSignals::UpdatedBlockTipSync(const CBlockIndex *pindexNew)
```

Comment:
Doesn't this also need to be called from `InvalidateBlock()`, potentially each time the tip is updated?